io.neovim.nvim.tool.tmux.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.neovim.nvim.tool.tmux
+branch: stable
+runtime: io.neovim.nvim
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: libevent
+    sources:
```

Comment:
as a matter of style `sources` is usually the last in a module.

Reply:
Fixed: https://github.com/flathub/flathub/pull/4119/commits/bdfa46f75a6fcfd8344b1bad938bb5a2b40dccc4